OrbitGl/App.cpp

```diff
@@ -176,15 +176,70 @@ void GLoadPdbAsync( const std::vector<std::wstring> & a_Modules )
     GModuleManager.LoadPdbAsync( a_Modules, [](){ GOrbitApp->OnPdbLoaded(); } );
 }
 
+//TODO: find a better name
+//-----------------------------------------------------------------------------
+void OrbitApp::StartRemoteCaptureBufferingThread()
+{
+    PRINT_FUNC;
+    m_MessageBufferThread = std::make_shared<std::thread>(&OrbitApp::ProcessBufferedCaptureData, this);
```

Comment:
Make sure buffers are empty. There might be data from previous runs leftover in the buffers.

Reply:
Thanks, should be fixed now!